### PR TITLE
simplified in-component logic

### PR DESCRIPTION
**Pull request by brtsmrtn**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,6 +15,7 @@
     "react-dom": "^17.0.1",
     "react-scripts": "4.0.1",
     "typescript": "^4.1.2",
+    "validator": "^13.5.2",
```
Comment (brtsmrtn) [inline package.json:18]: Preparation for URL validation update, it has however no function at all at the moment.

File: package.json [hunk 2]
```diff
@@ -45,6 +46,7 @@
     ]
   },
   "devDependencies": {
+    "@types/validator": "^13.1.3",
```
Comment (brtsmrtn) [inline package.json:49]: Preparation for URL validation update, it has however no function at all at the moment.

**Review by brtsmrtn — COMMENTED** [review]
Updated error messaging logic and added types to functions.
Changes in package.json/package-lock.json are unneccessary at the moment, they're just a leak from currently worked on URL validation solution.
Would you merge it @erikvanek, if satisfied?

**Comment by brtsmrtn** [comment]
@erikvanek, no need to check it anymore, I am merging...

File: src/components/Message.tsx [hunk 3]
```diff
@@ -1,14 +1,33 @@
...
-    return messageGenerator(this.props.url, this.props.items);
+    const { url, items } = this.props;
+    const errorMessage = errorMessageGenerator(url, items);
+    if (errorMessage) {
+      return <p>▶ {errorMessage}</p>;
+    } else {
+      if (items.length) {
+        return (
+          <p>
+            ▶ Wanna read something? You've got {items.length} link{""}
```
Comment (erikvanek) [inline src/components/Message.tsx:20]: This is cute <3

**Pull request merged**